src/app/project/comment-period/add-comment/add-comment.component.ts

```diff
@@ -143,15 +162,30 @@ export class AddCommentComponent implements OnInit, OnDestroy {
       );
   }
 
-  public onCancel() {
+  /**
+   * Handle when the user wants to leave the form and stop adding/editing a comment.
+   * Navigates the user away from the form if the user confirms they wish to leave.
+   * 
+   * @return {void}
+   */
+  public onCancel(): void {
     if (this.commentingMethod === 'externalEngagementTool') {
       this.router.navigate(['/p', this.currentProject._id, 'cp', this.commentPeriod._id]);
     } else if (confirm(`Are you sure you want to discard all changes?`)) {
       this.router.navigate(['/p', this.currentProject._id, 'cp', this.commentPeriod._id]);
     }
   }
 
-  public setEaoStatus(status: string) {
+  /**
+   * Handle setting the "response" to a comment. When adding the comment
+   * the user can publish, defer, or reject it immediately.
+   * 
+   * If no status is set by the user, set the status to "pending."
+   * 
+   * @param {string} status The comment status(publish, defer, reject).
```

Comment:
```suggestion
   * @param {string} status The comment status(publish, defer, reject, pending).
```
i would assume it could be pending as well according to your comment on line 183